ext/pgsql/tests/bug72197.phpt

```diff
@@ -31,6 +31,6 @@ pg_query($conn, "ROLLBACK");
 pg_close($conn);
 ?>
 --EXPECTF--
-pg_lo_create(): supplied resource is not a valid PostgreSQL link resource%w
```

Comment:
Ah, there was a `%w` here. Not sure why, but presumably putting that back will fix the windows failure.

Reply:
ohhh, I was also puzzled about this failure, but %w explains it!